edb/schema/pointers.py

```diff
@@ -1130,6 +1132,8 @@ def _process_create_or_alter_ast(
 
         This may be called in the context of either Create or Alter.
         """
+        from edb.edgeql.compiler import normalization as qlnorm
```

Comment:
Is there an import loop here?  You should be able to import the compiler modules normally.

Reply:
Hmm... it appears that there is no import loop, apparently that's a relic of some earlier refactoring.

Moved the import to the top.